RP.Automation.API/ApiTests.cs

```diff
@@ -0,0 +1,104 @@
+using FluentAssertions;
+using RP.Automation.API.Models;
+using RP.Automation.Core.Utilities;
+using System;
+using Xunit;
+
+namespace RP.Automation.API
+{
+    public class ApiTests : IDisposable
+    {
+        public DashboardService _dashboardService;
+        public WidgetService _widgetService;
+        public int _dashboardId;
+        public int _widgetId;
+
+        public ApiTests()
+        {
+            _dashboardService = new DashboardService();
+            _widgetService = new WidgetService();
+        }
+
+        [Fact]
+        public void CreateDashboard()
+        {       
+            var createDashboardData = new CreateDashboardData();
+            createDashboardData.Description = "description";
+            createDashboardData.Name = $"TestDashboard{Randomizer.RandomDigits()}";
+            createDashboardData.Share = true;
+
+            _dashboardId = _dashboardService.CreateDashboard(createDashboardData);
+            _dashboardId.Should().BeGreaterThan(0);
```

Comment:
U need to created Get All dashboards methods and compare the existing amount of dashboards